src/components/PostMeetingFollowUp.tsx

```
@@ -80,25 +98,122 @@ const PostMeetingFollowUp: React.FC<PostMeetingFollowUpProps> = ({
+        setEmailSubject(`Follow-up: ${generatedInsights.meetingName}`);
+      }
+      
+    } catch (err) {
+      console.error("Error generating insights:", err);
+      setError(err instanceof Error ? err.message : "Failed to generate meeting insights");
+    } finally {
+      setIsLoadingInsights(false);
+    }
+  };
+
+  useEffect(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on fetchMeetingData.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)